sensor_api/README.md

```diff
@@ -0,0 +1 @@
+PerfectFit API for sensor data
```

Comment:
Can you add some instructions on how to run the server?

Reply:
yep, it's added now.